### PR TITLE
fix: normalize every CRLF when parsing a RELEASES manifest

### PR DESCRIPTION
**Pull request by dopry**
`parseRELEASES` normalized line endings with `String.replace("\r\n", "\n")`, which only replaces the **first** occurrence — every subsequent line of a CRLF manifest kept its trailing `\r`. The line regex's `[\r]*$` tolerance made this harmless in practice, but the normalization now actually does what it says via `replaceAll` (string pattern, so the exec-spying edge-case test is unaffected).

Adds a multi-line CRLF parsing test.

From the 2.0 release review pass.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_01AtqCsX7ptP36t5rdyaJJhM

---
_Generated by [Claude Code](https://claude.ai/code/session_01AtqCsX7ptP36t5rdyaJJhM)_

### CODE REVIEW

File: src/utils/win-releases.ts [hunk 1]
```diff
@@ -72,7 +72,8 @@ export async function parseRELEASES(
   content: string,
 ): Promise<SquirrelRelease[]> {
   const stripped = stripBom(content);
-  const normalizedEOL = stripped.replace("\r\n", "\n");
+  // replaceAll: replace with a string pattern only replaces the first CRLF
+  const normalizedEOL = stripped.replaceAll("\r\n", "\n");
```
[the remark this answers is not in the document]
Comment (dopry) [reply src/utils/win-releases.ts]: Fixed in 48e4490 — reworded: "String#replace with a string pattern would only replace the first CRLF; replaceAll normalizes every line ending."

---
_Generated by [Claude Code](https://claude.ai/code)_

File: test/unit/win-releases.spec.ts [hunk 2]
```diff
@@ -44,6 +44,18 @@ describe("Windows RELEASES", function () {
...
+      const crlfReleases = await parseRELEASES(
+        "62E8BF432F29E8E08240910B85EDBF2D1A41EDF2 atom-0.178.0-full.nupkg 81272434\r\n" +
+          "5D754139E89802E88984185D2276B54DB730CD5E atom-0.178.1-delta.nupkg 8938535\r\n" +
+          "DD48D16EE177DD278F0A82CDDB72EBD043C767D2 atom-0.178.1-full.nupkg 81293415",
+      );
+      expect(crlfReleases.length).toBe(3);
+      // every CRLF is normalized, not just the first one
+      expect(crlfReleases[2].filename).toBe("atom-0.178.1-full.nupkg");
+      expect(crlfReleases[2].size).toBe(81293415);
+    });
```
[the remark this answers is not in the document]
Comment (dopry) [reply test/unit/win-releases.spec.ts]: Fixed in 48e4490: the test now captures every line handed to the regex via a RegExp#exec spy and asserts none still contains "\r". Verified it fails against the old first-only `replace` implementation and passes with `replaceAll`.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: test/unit/win-releases.spec.ts [hunk 3]
```diff
@@ -44,6 +44,35 @@ describe("Windows RELEASES", function () {
...
+        .mockImplementation(function (this: RegExp, str: string) {
+          seenLines.push(str);
+          return originalExec.call(this, str);
+        });
+      const crlfReleases = await parseRELEASES(
+        "62E8BF432F29E8E08240910B85EDBF2D1A41EDF2 atom-0.178.0-full.nupkg 81272434\r\n" +
+          "5D754139E89802E88984185D2276B54DB730CD5E atom-0.178.1-delta.nupkg 8938535\r\n" +
+          "DD48D16EE177DD278F0A82CDDB72EBD043C767D2 atom-0.178.1-full.nupkg 81293415",
+      );
+      spy.mockRestore();
```
[the remark this answers is not in the document]
Comment (dopry) [reply test/unit/win-releases.spec.ts]: Fixed in c259927: the spy now restores in a `finally` block, so a throwing `parseRELEASES` can't leak it into later tests.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: test/unit/win-releases.spec.ts [hunk 4]
```diff
@@ -44,6 +44,35 @@ describe("Windows RELEASES", function () {
...
+
+      expect(crlfReleases.length).toBe(3);
+      expect(crlfReleases[2].filename).toBe("atom-0.178.1-full.nupkg");
+      expect(crlfReleases[2].size).toBe(81293415);
+      // with first-only replacement the second and third lines would still
+      // end in \r when they reach the regex
```
[the remark this answers is not in the document]
Comment (dopry) [reply test/unit/win-releases.spec.ts]: Fixed in c259927: the comment now says only the second (CRLF-terminated) line would retain a CR under first-only replacement, noting the last line has no CRLF terminator. Also tightened the source comment to say "every CRLF occurrence" rather than "every line ending".

---
_Generated by [Claude Code](https://claude.ai/code)_
